backend/src/test/groovy/com/capstone/moneytree/model/SanitizedUserTest.groovy

```diff
@@ -0,0 +1,17 @@
+package com.capstone.moneytree.model
+import com.capstone.moneytree.model.node.User
+import static com.capstone.moneytree.utils.MoneyTreeTestUtils.*
+import spock.lang.Specification
+
+/**
+ * Unit Tests for the SanitizedUser model.
+ */
+class SanitizedUserTest extends Specification {
+    private static SanitizedUser sanitizedUser;
+
+    def setup() {
+        User user = createUser(1, "test@test.com", "username", "password", "Jake", "Moreau", "123B")
+        sanitizedUser = new SanitizedUser(user)
```

Comment:
Did you do a test inside this Groovy class? I don't see one. Maybe my browser is being weird.

Reply:
My bad, I forgot that, just added it in my new commit.